### DISCUSSION TITLE
Is there any API available for Glassdoor or Monster to post jobs from a web application?

### DISCUSSION THREAD

**Discussion by Pola5h** [community/community#131763] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

I have a web application and want to post jobs on Monster and Glassdoor from my application. Is there an API available for this?

I was looking for an API but can't find any related articles or resources. I need detailed information or documentation on how to achieve this integration. Can anyone provide guidance or point me to the right resources?

**Comment by harshjuly12** [comment]
Hey try this: https://recruiterswebsites.com/app/monster/

**Comment by samus-aran** [comment]
Hi @Pola5h 👋 

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this website/service, such as the Monster/Glassdoor community. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
